### DISCUSSION TITLE
blacklist for macaque species

### DISCUSSION THREAD

**Discussion by sandiegolei** [stuart-lab/signac#1416] [Q&A] [answered]
Hi,
I'm using Signac to analyze ATAC single cell data from macaque.
Following the steps provided in the Signac, I need to use subsetByOverlaps to remove the peaks listed in the blacklist like for human.
Therefore, I want to ask how to get the blacklist for a new species like here macaque mulatta mmul_10? Could you give some suggestions for meeting such a new species? Can I go ahead analysis by skipping this step?
Many many thanks,
lei

**Comment by timoast** [comment, marked as answer]
I'd suggest taking a look at the methods used to create the ENCODE blacklists for the other species, maybe you can try applying that to macaque: https://www.nature.com/articles/s41598-019-45839-z
